kernel/Makefile

```diff
@@ -44,4 +47,6 @@ clean:
 	rm $(CXX_OBJS)
 	rm $(AS_OBJS)
 	rm $(ELF_IMAGE)
+	rm $(DEP)
 
+-include $(DEP)
```

Comment:
;)

Reply:
https://www.youtube.com/watch?v=xquR0za0WXw

Reply:
I'll destroy you